security/medusa/l1/medusa.c

```diff
@@ -449,7 +452,6 @@ static int medusa_l1_path_symlink(const struct path *dir, struct dentry *dentry,
 static int medusa_l1_path_link(struct dentry *old_dentry, const struct path *new_dir,
 			 struct dentry *new_dentry)
 {
-
 	return validate_fuck_link(old_dentry);
 }
 
```

Comment:
should be:
`if (validate_fuck_link(old_dentry) == MED_NO)
        return -EACCESS;`

Reply:
This will be fixed as part of kobject_fuck refactoring.